kernelci/rootfs.py

```diff
@@ -0,0 +1,41 @@
+# Copyright (C) 2019 Collabora Limited
+# Author: Lakshmipathi G <lakshmipathi.ganapathi@collabora.com>
+#
+# This module is free software; you can redistribute it and/or modify it under
+# the terms of the GNU Lesser General Public License as published by the Free
+# Software Foundation; either version 2.1 of the License, or (at your option)
+# any later version.
+#
+# This library is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more
+# details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this library; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+
+from kernelci import shell_cmd
+
+
+def build(name, config, debos_path):
```

Comment:
This function is specific to `debos`.  It should instead look at the rootfs type from the config, and call a `_build_debos()` function or similar if it's `debos` or return an error if it's anything else.  We'll be adding `_build_buildroot()` or something like that once the implementation around `debos` is complete.

Reply:
okay, added this change and updated the PR